src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml

```diff
@@ -1534,11 +1534,6 @@
 											<DiscreteObjectKeyFrame KeyTime="0"
 																	Value="Transparent" />
 										</ObjectAnimationUsingKeyFrames>
-										<ObjectAnimationUsingKeyFrames Storyboard.TargetName="MultiSelectSquare"
-																	   Storyboard.TargetProperty="Background">
-											<DiscreteObjectKeyFrame KeyTime="0"
-																	Value="{ThemeResource SystemControlHighlightAccentBrush}" />
```

Comment:
Let's keep all those animations

Reply:
I can use the following way:
```C#
	if (Selector is ListView lv)
	{
		ApplyMultiSelectState(lv.IsSelectionMultiple);
	}
	else if (Selector is GridView gv)
	{
		ApplyMultiSelectState(gv.IsSelectionMultiple);
	}
```
But this violates the "Open-Closed principle".
With this implementation, I will have to change **OnLoaded** method every time if I have new elements for the multi-select function, or new functionality.
Consequently, I have a higher chance of getting a conflict when merging into a master/main branch.

A more correct solution would be to place this functionality for each SelectorItem type.

Reply:
Not a problem, but its will look like this:
![image](https://github.com/unoplatform/uno/assets/69314237/fec7e458-213d-4405-94fb-e2161abe6f50)

I removed this animations to make similar as in ListViewItem

https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml#L1132-L1137

Reply:
Fixed
![Screenshot 2023-09-08 110742](https://github.com/unoplatform/uno/assets/69314237/0e1db728-01c8-4008-ace9-cd3f10135479)